spring-core-5/src/test/java/com/baeldung/springbean/naming/SpringBeanNamingUnitTest.java

```diff
@@ -0,0 +1,70 @@
+package com.baeldung.springbean.naming;
+
+import com.baeldung.springbean.naming.service.PetShow;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(SpringExtension.class)
+public class SpringBeanNamingUnitTest {
+
+    private AnnotationConfigApplicationContext context;
+
+    @BeforeEach
+    void setUp() {
+        context = new AnnotationConfigApplicationContext();
+        context.scan("com.baeldung.springbean.naming");
+        context.refresh();
+    }
+
+    //    To name a bean spring gets the class name and converts the first letter to lowercase. 
```

Comment:
Minor comment on the formatting. No need for tabs in the comments. One space is enough. See our own eclipse formatter for example - https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml

Reply:
fixed